src/modules/Player/components/TimeRange/TimeRange.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 
-import { useStore } from '@/store/index';
+import { useStore } from '@/store';
 
 import format from '@/modules/Player/helpers/format';
 
+import InputRange from '@/ui/Input/inputRange/InputRange.tsx';
```

Comment:
путь не оттуда подтянул
```suggestion
import InputRange from '@/ui/Input';
```